RTCError and RTCErrorEvent docs (#44047)
mdn/content: files/en-us/web/api/rtcerror/sdplinenumber/index.md

{{APIRef("WebRTC")}}

> The **`sdpLineNumber`** read-only property of the {{domxref("RTCError")}} interface specifies the line number within the {{Glossary("SDP")}} at which a syntax error occurred while parsing it.

Reviewer: ```suggestion
The **`sdpLineNumber`** read-only property of the {{domxref("RTCError")}} interface specifies the {{Glossary("SDP")}} message line number where a syntax error occurred.
```

## Value

> An integer value indicating the line within the SDP at which the syntax error described by the `RTCError` object occurred.

Reviewer: ```suggestion
An integer value indicating the SDP message line number where the syntax error described by the `RTCError` object occurred.
```
